resources/assets/js/components/Modal.tsx

```diff
@@ -1,189 +1,168 @@
-import React, { Component, createRef } from "react";
```

Comment:
Meta comment about branch/PR hygene - maybe the changes to the Modal should (ideally) be its own PR

Reply:
It probably should be, although it was driven by the functionality in this story so I thought it was alright in this case.

Reply:
Yeah, I do this all the time, and I don't expect you to separate it out now. But with this bout of code reviews I've been wondering how we can improve our PR/branch strategies. And I think small feature branches that don't depend on anything except dev, that can be accepted quickly and easily, is the way to go.